go.mod

```diff
@@ -3,26 +3,28 @@ module github.com/k8snetworkplumbingwg/sriov-cni
 go 1.19
 
 require (
-	github.com/containernetworking/cni v1.1.2
-	github.com/containernetworking/plugins v1.1.1
-	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.17.0
-	github.com/stretchr/testify v1.5.1
-	github.com/vishvananda/netlink v1.1.1-0.20210330154013-f5de75959ad5
-	golang.org/x/net v0.7.0
+	github.com/containernetworking/cni v1.1.3-0.20230206011450-121798a08dcc
+	github.com/containernetworking/plugins v1.2.1-0.20230314104937-3bc00017e302
```

Comment:
not a fan of bumping those two (cni and plugins) to non released version.

if those drag in the problematic dependency, we can overwrite in a replace directive [1] WDYT? 

[1] https://go.dev/ref/mod#go-mod-file-replace

Reply:
Hi @adrianchiris, thanks for the review. Makes sense to me, just pushed these changes.